framework/include/base/Assembly.h

```diff
@@ -41,8 +41,10 @@ class MooseMesh;
 class ArbitraryQuadrature;
 class SystemBase;
 class MooseVariableFEBase;
+class MooseVariableBase;
 template <typename>
 class MooseVariableFE;
+class MooseVariableScalar;
```

Comment:
I'm surprised these weren't already included or forward declared?

Reply:
Yes. Almost everything in Assembly.h already works this way. I would like to jus follow them.